src/richie/apps/courses/templates/courses/cms/fragment_course_glimpse.html

```diff
@@ -1,13 +1,14 @@
 {% load i18n cms_tags extra_tags static thumbnail %}{% spaceless %}
 {% comment %}Obviously, the context template variable "course" is required and must be a Course page extension{% endcomment %}
 
-{% with course_page=course.extended_object course_state=course.state main_org_title=course.get_main_organization.extended_object.get_title course_variant=course_variant|default:"glimpse" %}
-<a class="course-{{ course_variant }}{% if course_page.publisher_is_draft is True %} course-{{ course_variant }}--draft{% endif %}"
- href="{{ course_page.get_absolute_url }}"
->
-    <div class="course-{{ course_variant }}__media">
+{% with course_page=course.extended_object course_state=course.state main_org_title=course.get_main_organization.extended_object.get_title course_variant=course_variant %}
+<a class="course-glimpse  
+    {% if course_variant is not None %} course-glimpse--{{ course_variant }}{% endif %}
+    {% if course_page.publisher_is_draft is True %} course-glimpse--draft{% endif %}"
+    href="{{ course_page.get_absolute_url }}">
+    <div class="course-glimpse__media">
         {% get_page_plugins "course_cover" course_page as cover_plugins or %}
-            <p class="course-{{ course_variant }}__empty">{% trans "Cover" %}</p>
+            <p class="course-glimpse__empty">{% trans "Cover" %}</p>
```

Comment:
should it be --empty ? it is somewhere else in the code...

Reply:
Good catch. Yep a wrong class name was used.